### PR TITLE
Amend OS/Kernel version reporting to include pretty o/s and kernel version.

### LINKED ISSUE #3010: Add O/S Kernel version to the SBOM
In addition to capturing the OS_FULL_VERSION, it would be worth adding a seperate section to capture the O/S Kernel version, particulary in the context of docker containers, where the underlying O/S kernel may be different to the O/S.

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes #3010

### CODE REVIEW

**Review by karianna — CHANGES_REQUESTED** [review]
Whitespace seems incorrect?

File: sbin/common/config_init.sh [hunk 1]
```diff
@@ -391,32 +391,22 @@ function setBranch() {
 # Set the config defaults
 function configDefaults() {
 
-  # The OS kernel name, e.g. 'darwin' for Mac OS X
-  BUILD_CONFIG[OS_KERNEL_NAME]=$(uname | awk '{print tolower($0)}')
+# The OS kernel name, e.g. 'darwin' for Mac OS X
```
Comment (karianna) [inline sbin/common/config_init.sh:394]:
```suggestion
  # The OS kernel name, e.g. 'darwin' for Mac OS X
```

File: sbin/common/config_init.sh [hunk 2]
```diff
@@ -391,32 +391,22 @@ function setBranch() {
...
+BUILD_CONFIG[OS_KERNEL_NAME]=$(uname | awk '{print tolower($0)}')
```
Comment (karianna) [inline sbin/common/config_init.sh:395]:
```suggestion
  BUILD_CONFIG[OS_KERNEL_NAME]=$(uname | awk '{print tolower($0)}')
```

**Review by karianna — CHANGES_REQUESTED** [review]
suggested some more whitespace fixes

**Review by zdtsw — APPROVED** [review]
LGTM

**Pull request merged**